### PR TITLE
0.6.0 update

### PR DESCRIPTION
**Pull request by David-OConnor**
@MartinKavik is there anything else we need to change, ie to update the docs links?

### CODE REVIEW

File: crate/guides/changelog.md [hunk 1]
```diff
@@ -1,11 +1,37 @@
 # Changelog
 
-# Changelog
-
 [unreleased]
-
 - (placeholder)
 
+## v0.6.1
```
Comment (MartinKavik) [inline crate/guides/changelog.md]: 0.6.1 vs 0.6.0

File: crate/src/lib.rs [hunk 2]
```diff
@@ -21,8 +21,7 @@ use serde_json;
 use std::{convert::identity, fmt};
 use Visibility::*;
 
-const SEED_VERSION: &str = "0.5.1";
-const SEED_VERSION_DATE: &str = "Dec 28, 2019";
+const SEED_VERSION: &str = "0.6.0 (February 1, 2020)";
```
Comment (MartinKavik) [inline crate/src/lib.rs]: I've split it into two variables in this PR https://github.com/seed-rs/seed-rs.org/pull/23 for more flexible styling

**Comment by MartinKavik** [comment]
- You know docs more than I do (i.e. if content corresponds to new changes in changelog) and it looks like you done all steps related to this repo.
- I don't have time to write/find linter for dead links (https://github.com/seed-rs/seed-rs.org/issues/17).
- I don't want to add any other PRs.
- So from my point of view it's ready.

**Comment by David-OConnor** [comment]
Sounds good. I was holding off for a chance to write the migration guide, but we can shelve that for this version, since I agree with you, and think the changelog's sufficient.

**Pull request merged**